upload-artifact.gradle

```diff
@@ -1,5 +1,35 @@
 apply plugin: 'com.bmuschko.nexus'
 apply plugin: 'signing'
+
+modifyPom {
```

Comment:
Can you explain to me quickly what this part is for?

Reply:
this modifyPom part is to build a POM XML file that meets the maven standard of released packages, a POM XML file should contain the information of project and configuration information for the maven to build the project